test/benchmark/vqe_application.py

```diff
@@ -40,34 +33,82 @@ def __init__(self,
-            initial_state = HartreeFock(num_orbitals, num_particles, qubit_mapping,
-                                        two_qubit_reduction, z2_symmetries.sq_list)
-            var_form = UCCSD(num_orbitals=num_orbitals,
-                             num_particles=num_particles,
-                             initial_state=initial_state,
-                             qubit_mapping=qubit_mapping,
-                             two_qubit_reduction=two_qubit_reduction,
-                             z2_symmetries=z2_symmetries)
-            vqe = VQE(var_form=var_form, include_custom=True, optimizer=SLSQP(maxiter=5000), max_evals_grouped=256)
-            vqe.quantum_instance = Aer.get_backend('qasm_simulator')
-            vqe.quantum_instance.backend_options['backend_options'] = {'max_parallel_experiments':threads, 'method': method} 
-            
-            return vqe
-        mgse = MolecularGroundStateEnergy(driver)
-        result = mgse.compute_energy(cb_create_solver)
-            
+    def _run_uccsd_vqe(self, mol_string, method, simulator, threads):
+        
+        from qiskit_nature.circuit.library import HartreeFock, UCCSD
+        from qiskit_nature.converters.second_quantization import QubitConverter
+        from qiskit_nature.drivers import UnitsType, Molecule
+        from qiskit_nature.drivers.second_quantization.pyscfd import PySCFDriver
+        from qiskit_nature.mappers.second_quantization import JordanWignerMapper, ParityMapper
+        from qiskit_nature.problems.second_quantization import ElectronicStructureProblem
+        
+        driver = PySCFDriver(atom=mol_string, unit=UnitsType.ANGSTROM, basis='sto3g')
+        es_problem = ElectronicStructureProblem(driver)
+        qubit_converter = QubitConverter(JordanWignerMapper())
+        max_evals_grouped = 1024
+        
+        from qiskit.algorithms.optimizers import SLSQP
+        optimizer = SLSQP(maxiter=5000)
+        
+        from qiskit.utils import QuantumInstance
+        
+        quantum_instance = QuantumInstance(backend=simulator)
+        
+        # import logging
+        # logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s',
+        #                     level=logging.DEBUG,
+        #                     datefmt='%Y-%m-%d %H:%M:%S')
+        
+        from qiskit_nature.algorithms import VQEUCCFactory
+        
+        vqe_solver = VQEUCCFactory(quantum_instance,
+                                   optimizer=optimizer,
+                                   include_custom=True,
+                                   max_evals_grouped=max_evals_grouped)
+        
+        from qiskit_nature.algorithms import GroundStateEigensolver
+        
+        calc = GroundStateEigensolver(qubit_converter, vqe_solver)
+        
+        res = calc.solve(es_problem)
+        
     def time_statevector(self, mol_name):
+        from qiskit.providers.aer import AerSimulator
```

Comment:
We definitely don't want to be running imports in measured methods since it will skew the runtime especially if multiple trials are done.